script.js

```diff
@@ -69,4 +69,22 @@ function alignText(elem, alignType){
     buttonList[i].classList.remove('active');
   }
   elem.classList.add('active');
-}
\ No newline at end of file
+}
+
+function fontSize(elem, sizeOfFont) {
+  document.getElementById('text-output').style.fontSize = sizeOfFont;
+  let buttonList = document.getElementsByClassName('size');
+  for (let i=0; i < buttonList.length; i++) {
+    buttonList[i].classList.remove('active');
+  }
+  elem.classList.add('active');
+}
+
+function changeColor() {
+  document.getElementById('text-output').style.color = document.getElementById("ColorPickerName").value;
+}
+
+function textToSpeech() {
+  var msg = new SpeechSynthesisUtterance(document.getElementById('text-input').value);
+  window.speechSynthesis.speak(msg);
+}
```

Comment:
- [ ] Add functionality to change different voices and set a specific default voice on start.